packages/manager/modules/pci/src/projects/project/storages/databases/components/input-rule/index.js

```diff
@@ -0,0 +1,13 @@
+import angular from 'angular';
+import '@ovh-ux/ui-kit';
+
+import component from './input-rule.component';
+
+const moduleName = 'ovhManagerPciStoragesDatabasesInputRule';
+
+angular
+  .module(moduleName, ['oui'])
+  .component('inputRule', component)
```

Comment:
Name is a bit generic for a scoped component

Reply:
Would `pciInputRule` be better name to have it scoped to the pci universe ?

Reply:
Maybe still too generic as we are in /storages/databases, something like `pciStorageDatabaseInputRule`

Reply:
Changed here: https://github.com/ovh/manager/pull/6263/commits/c1d3c5629b9225d2f11784fb88688fc9b4389a2d